src/components/picker/helpers/usePickerMigrationWarnings.tsx

```diff
@@ -1,10 +1,10 @@
 import {useEffect} from 'react';
 import {LogService} from '../../../services';
-import {PickerProps} from '../types';
+import {PickerPropsDeprecation} from '../types';
 
 // TODO: Remove this whole file when migration is completed
 type UsePickerMigrationWarnings = Pick<
-  PickerProps,
+PickerPropsDeprecation,
```

Comment:
Why didn't you add warnings to the new deprecated props?

Reply:
We had some issue with some of the props that are still in use in our internal code, I'll do it in separate PR.